open-craft/problem-builder
# Step Builder: Review step and assessment functionality

This PR introduces a `ReviewStep` block and adds assessment functionality to the new mentoring block (Step Builder).

**Testing**

1. Enable Step Builder by adding `step-builder` to the list of advanced modules in Studio.
2. Create a new unit and add a Step Builder block.
3. Add a few Mentoring Step blocks to the Step Builder block.
4. Add a Review Step block to the Step Builder block. Note that you can only add a single Review Step block (after adding it the corresponding button becomes disabled).
5. Add some questions to each step.
6. Test functionality by completing the questions you added in the LMS, trying different settings for `max_attempts` and `extended_feedback`.
